.sass-lint.yml

```diff
@@ -0,0 +1,347 @@
+# sass-lint config generated by make-sass-lint-config v0.1.2
+# 
+# The following scss-lint Linters are not yet supported by sass-lint:
+# ElsePlacement, PropertyCount, SelectorDepth, UnnecessaryParentReference
+# 
+# The following settings/values are unsupported by sass-lint:
+# Linter Indentation, option "allow_non_nested_indentation"
+# Linter Indentation, option "character"
+# Linter PropertySortOrder, option "separate_groups"
+# Linter SpaceBeforeBrace, option "allow_single_line_padding"
+
+files:
+  include: scss/**/*.scss
+options:
+  formatter: stylish
+  merge-default-rules: false
+rules:
+  border-zero:
+    - 1
+    - convention: zero
+  brace-style:
+    - 1
+    - allow-single-line: false
+  class-name-format:
+    - 1
+    - convention: '([a-z0-9]+-?)+'
+  clean-import-paths:
+    - 1
+    - filename-extension: false
+      leading-underscore: false
+  empty-line-between-blocks:
+    - 1
+    - ignore-single-line-rulesets: true
+  extends-before-declarations: 1
+  extends-before-mixins: 1
+  final-newline:
+    - 1
+    - include: true
+  force-attribute-nesting: 1
+  force-element-nesting: 1
+  force-pseudo-nesting: 1
+  function-name-format:
+    - 1
+    - allow-leading-underscore: true
+      convention: hyphenatedlowercase
+  hex-length:
+    - 1
+    - style: short
+  hex-notation:
+    - 1
+    - style: lowercase
+  id-name-format:
+    - 1
+    - convention: '([a-z0-9]+-?)+'
+  indentation:
+    - 1
+    - size: 2
+  leading-zero:
+    - 1
+    - include: true
+  mixin-name-format:
+    - 1
+    - allow-leading-underscore: true
+      convention: hyphenatedlowercase
+  mixins-before-declarations: 1
+  nesting-depth:
+    - 1
+    - max-depth: 3
+  no-color-keywords: 1
+  no-color-literals: 1
+  no-css-comments: 0
+  no-debug: 1
+  no-duplicate-properties: 1
+  no-empty-rulesets: 1
+  no-ids: 1
+  no-important: 0
+  no-invalid-hex: 1
+  no-mergeable-selectors: 1
+  no-misspelled-properties:
+    - 1
+    - extra-properties: []
+  no-qualifying-elements:
+    - 1
+    - allow-element-with-attribute: false
+      allow-element-with-class: false
+      allow-element-with-id: false
+  no-trailing-zero: 1
+  no-url-protocols: 1
+  no-vendor-prefixes: 1
+  placeholder-in-extend: 1
+  placeholder-name-format:
+    - 1
+    - convention: '([a-z0-9]+-?)+'
+  property-sort-order:
+    - 1
+    - ignore-custom-properties: false
+    - 
+      order :
+        # Position
+        - position
```

Comment:
Their internal config files use quotes : https://github.com/sasstools/sass-lint/tree/develop/lib/config/property-sort-orders

I think it is better because of properties are following *external* naming rules (CSS), and not YML ones. Even if in practice it don't change anything.